toolbox/__init__.py

```diff
@@ -212,6 +237,20 @@ def vocoder_progress(i, seq_len, b_size, gen_rate):
         wav = wav / np.abs(wav).max() * 0.97
         self.ui.play(wav, Synthesizer.sample_rate)
 
+        #Enable replay and save buttons:
+        self.ui.replay_wav_button.setDisabled(False)
+        self.ui.save_wav_button.setDisabled(False)
+
+        #Update waves combobox
+        self.current_wav = wav
+        if len(self.waves_list) == MAX_WAVES:
+          self.waves_list.pop(0)          
+        self.waves_list.append(wav)
+        n_waves=len(self.waves_list)
+        #TODO better naming for the combobox items?
+        self.ui.waves_list_model.setStringList(["%d" % (i+1) for i in range(n_waves)])
```

Comment:
Please use a list of strings for the combobox items. This will provide a foundation for the TODO.

Also, instead of using the index to generate the stringList, can we instead use the count add continue incrementing even when MAX_WAVES is exceeded? For example when 1-15 already exist, the next vocode will add "16" to the list and drop "1" to make room. Otherwise they will still be called 1-15 but from the perspective of the user, all of the remaining waves have shifted by 1 which is very confusing.

Lastly, it would be more intuitive to display the most recently generated wav at the top. Can you please work this in? I also don't mind submitting a PR to your fork if that's what you prefer.

Reply:
Okay, I inverted the combobox order and kept the numbering which makes sense. 

> Please use a list of strings for the combobox items. This will provide a foundation for the TODO.

Not sure what exactly you mean with this but take a look on my last commit.

>  I also don't mind submitting a PR to your fork if that's what you prefer.

Sure that is fine!